gradio/static/js/gradio.js

```diff
@@ -317,6 +380,82 @@ function gradio(config, fn, target, example_file_path) {
     })
     target.find(".load_prev").click(prev_example);
     target.find(".load_next").click(next_example);
+    target.find(".order_similar").click(function() {
+      io_master.score_similarity(function() {
+        io_master.current_page = 0
+        io_master.current_example = null;
+        load_page();  
+      })
+    });
+    target.find(".view_examples").click(function() {
+      target.find(".examples > table").removeClass("invisible");        
```

Comment:
can you attach a second class to all example related elements called "example-content", and all embedded related content called "embedding-content", and just toggle the visibility of this secondary class? instead of targeting every single element